addons/account/partner.py

```diff
@@ -254,9 +256,63 @@ def _journal_item_count(self):
             partner.contracts_count = self.env['account.analytic.account'].search_count([('partner_id', '=', partner.id)])
 
     @api.multi
+    def _compute_issued_total(self):
+        for partner in self:
+            issued_total = 0
+            domain = [('partner_id', '=', partner.id), ('reconciled', '=', False), ('account_id.deprecated', '=', False), ('account_id.internal_type', '=', 'receivable')]
+            for aml in self.env['account.move.line'].search(domain):
+                overdue = aml.date_maturity and datetime.today().strftime('%Y-%m-%d') > aml.date_maturity
+                if overdue:
+                    issued_total += aml.amount_residual
+            partner.issued_total = formatLang(self.env, issued_total, currency_obj=self.env.user.company_id.currency_id)
+
+    @api.multi
     def mark_as_reconciled(self):
         return self.write({'last_time_entries_checked': time.strftime(DEFAULT_SERVER_DATETIME_FORMAT)})
 
+    def get_partners_in_need_of_action(self):
```

Comment:
get_partners_in_need_of_action() and get_partners_with_overdue() need refactoring...